angular/src/app/containers/leaflet/leaflet.component.ts

```diff
@@ -29,11 +47,76 @@ export class LeafletComponent implements AfterViewInit {
         bounds: config.maxBounds
       });
 
-      baseLayer.addTo(leafletMap);
+      baseLayer.addTo(this.leafletMap);
 
       setTimeout(() => {
-        leafletMap.invalidateSize();
+        this.leafletMap.invalidateSize();
       });
+
+      this.leafletMap.on('moveend', this.updateBoundingBox.bind(this));
```

Comment:
Ja, dat was wat ik bedoelde met de externe library, want dat meegeven is ook niet heel fraai natuurlijk. Misschien even zo laten en later eens onderzoeken hoe dat bijvoorbeeld in React apps gedaan wordt.